modules/data-entry/src/main/frontend/src/dataHomepage/ExportButton.jsx

```diff
@@ -0,0 +1,383 @@
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+import React, { useState, useEffect } from 'react';
+import PropTypes from "prop-types";
+import { makeStyles } from '@mui/styles';
+import { deepPurple, orange } from '@mui/material/colors';
+import { Avatar, Checkbox, Dialog, DialogTitle, DialogActions, DialogContent, FormControl, Icon, Grid, Radio, RadioGroup,
+  FormControlLabel, Typography, Button, IconButton, Tooltip, InputLabel, Select, ListItemText, MenuItem } from "@mui/material";
+import { FileDownload } from "@mui/icons-material";
+import CloseIcon from '@mui/icons-material/Close';
+
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    "& .MuiGrid-item:not(:first-child)": {
+      paddingTop: theme.spacing(2),
+    },
+  },
+  entryContainer: {
+    paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(2),
+  },
+  entryActionIcon: {
+    float: "right",
+    marginRight: theme.spacing(1),
+  },
+  variableDropdown: {
+    "& > .MuiInputLabel-root" : {
+      maxWidth: `calc(100% - ${theme.spacing(3)})`,
+    },
+  },
+  variableOption: {
+    whiteSpace: "normal",
+  },
+  valueEntry: {
+    border: "1px solid " + theme.palette.divider,
+    borderRadius: theme.spacing(.5, 3, 3, .5),
+    margin: theme.spacing(1, 0),
+    "& > .MuiGrid-item" : {
+      display: "flex",
+      paddingLeft: theme.spacing(1.5),
+      alignItems: "center",
+    },
+  },
+  valueActions: {
+    justifyContent: "flex-end",
+    paddingTop: "0!important"
+  },
+  avatar: {
+    marginRight: theme.spacing(1),
+    opacity: '50%'
```

Comment:
I think we can drop the opacity setting if we make the avatar smaller

Reply:
I would also align the avatar to the top, wdyt?
```suggestion
    marginTop:  theme.spacing(1),
    alignSelf: "start",
```